<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound Setup</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QPushButton" name="testSoundButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>103</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Testsound</string>
        </property>
       </widget>
       <widget class="QPushButton" name="closeButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>103</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopSoundButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Test</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QSlider" name="volumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Volume</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
